requests/models.py

```diff
@@ -870,4 +870,6 @@ def close(self):
         if not self._content_consumed:
             self.raw.close()
 
-        return self.raw.release_conn()
+        release_conn = getattr(self.raw, 'release_conn', None)
+        if release_conn is not None:
+            return release_conn()
```

Comment:
No we don't, I was just trying to change as little of the existing code as needed. I'm fine with removing it.